src/components/sendWritable/send.js

```diff
@@ -93,14 +95,23 @@ class SendWritable extends React.Component {
           <TransferTabs setTabSend={this.props.setTabSend} isActiveTabSend={true}/>
         </div>
         <form className={styles.form}>
-          <AddressInput
-            className='recipient'
-            label={this.props.t('Send to address')}
-            address={this.state.recipient}
-            handleChange={this.handleChange.bind(this, 'recipient', true)}
-          />
+          { this.props.followedAccounts.length > 0 && this.state.openFollowedAccountSuggestion ?
+            <Bookmark
+              focusReference={this.focusReference.bind(this)}
+              className='recipient'
+              label={this.props.t('Send to address')}
+              address={this.state.recipient}
+              handleChange={this.handleChange.bind(this, 'recipient', true)}
+            /> :
+            <AddressInput
+              className='recipient'
+              label={this.props.t('Send to address')}
+              address={this.state.recipient}
+              handleChange={this.handleChange.bind(this, 'recipient', true)}
+            />
+          }
           <ReferenceInput
-            className='reference'
+            context={this}
             label={this.props.t('Reference (optional)')}
             address={this.state.reference}
```

Comment:
Please fix the param name `address={this.state.reference}` -> `reference={this.state.reference}`. I get it's not related to this PR, but it's a small refactor not worth its own PR.